src/containers/MemberCreationPage.tsx

```diff
@@ -0,0 +1,172 @@
+}) => {
+  const history = useHistory();
+
+  const { data, loading } = useQuery(
+    mode === 'edit' && userId ? GET_USER_BY_ID(userId) : GET_ALL_USERS
+  );
+
+  const users: Array<User> = data ? data.users : [];
+
+  const [addUser] = useMutation(ADD_USER, {
+    update(cache, { data: { newUser } }) {
+      cache.writeQuery({
+        query: GET_ALL_USERS,
+        data: { users: users.concat([newUser]) },
+      });
+    },
+  });
+  const [editUser] = useMutation(EDIT_USER);
+
+  const [memberName, setMemberName] = useState<string>('');
+  const [email, setEmail] = useState<string>('');
+  const [role, setRole] = useState<AccessLevel>(AccessLevel.SUPERVISOR);
+
+  useEffect(() => {
+    if (!loading && mode === 'edit') {
+      const {
+        name,
+        email,
+        accessLevel,
+      }: {
+        name: string;
+        email: string;
+        accessLevel: AccessLevel;
+      } = data.user;
+      setMemberName(name);
+      setEmail(email);
+      setRole(accessLevel);
+    }
+  }, [data]);
+
+  const handleNameChange = (e: any) => {
+    setMemberName(e.target.value);
+  };
+
+  const handleEmailChange = (e: any) => {
+    setEmail(e.target.value);
+  };
+
+  const handleRoleChange = (e: any) => {
+    setRole(e.target.value);
+  };
+
+  const handleComplete = () => {
+    if (mode === 'new') {
+      addUser({
+        variables: {
+          name: memberName,
+          email,
+          password: 'password',
+          accessLevel: role,
+          emergencyContact: '1234567890',
+        },
+      });
+    } else if (mode === 'edit') {
+      editUser({
+        variables: {
+          id: userId,
+          name: memberName,
+          email,
+          accessLevel: role,
+        },
+      });
+    }
+    history.replace('/manage');
+  };
+
+  return (
+    <div className="resource-add-wrapper">
+      <div className="resource-creation-top-section">
+        <BackLink to="/manage" />
+        <div className="resource-header">
+          <Typography variant="h4">
+            {mode === 'new' ? 'Add a new team member' : 'Edit team member'}
+          </Typography>
+        </div>
+        {mode === 'new' ? (
+          <div className="top-bar-link">
+            <Typography
+              variant="caption"
+              style={{ color: Colours.SecondaryGray }}
+            >
+              New team members will receive sign up instructions via email.
+            </Typography>
+          </div>
+        ) : (
+            ''
+          )}
+      </div>
+      <ValidatorForm onSubmit={handleComplete}>
+        <div className="event-form">
+          <FormField
+            label="Team Member Name:"
+            required
+            isValidated={false}
+            onChange={handleNameChange}
+            value={memberName}
+          />
+          <FormField
+            label="Email:"
+            required
+            // eslint-disable-next-line react/jsx-boolean-value
+            isValidated={true}
```

Comment:
can't this be resolved by just setting `isValidated` and not assigning it a value? (i.e. like `required` 2 lines above)